setup.cfg

```diff
@@ -32,15 +32,15 @@ packages = find:
 python_requires = >=3.7
 zip_safe = False
 install_requires =
-    six>=1.10
+    importlib-metadata>=5.0  # remove when Python >=3.12
     isbnlib>=3.10.8
 
 [options.extras_require]
 tests =
     pytest-black-ng>=0.4.0
     pytest-cache>=1.0
+    pytest-invenio>=1.4.0  # required for fixtures
```

Comment:
@tmorrell I had to put it back, @zzacharo very wisely suggested that it was necessary for the [entry_points](https://github.com/inveniosoftware/pytest-invenio/blob/d03d0b541a4e74c57a0437150a34329ae0716246/pytest_invenio/fixtures.py#L810) fixtures (and there is a lot code there).

Reply:
Yup, the thought was we could pull out the entry point code....but probably not worth the effort.